packages/valory/connections/test_libp2p/tests/test_dht.py

```diff
@@ -281,3 +304,32 @@ def teardown(self):
         """Clean up after test case run."""
         self.unset_agent_context()
         self.run_cli_command("delete", self.agent_name)
+
+
+test_classes = [
+    Libp2pConnectionDHTRelay,
+    Libp2pConnectionDHTDelegate,
+    Libp2pConnectionDHTRelayAEACli,
+    Libp2pConnectionDHTDelegateAEACli,
+]
+
+
+@dataclass
+class TestCaseConfig:
+    """TestCase"""
+
+    name: str
+    nodes: List[NodeConfig]
+    start_local: bool = False
+
+
+# dynamically create tests
+for test_cls in test_classes:
```

Comment:
Fine for the staging and prod ones which we run locally only. But for the ones running in CI (those with local nodes) please have static test cases

Reply:
I have fixed it another way now, using an environmental variable and a pytest marker